packages/sheets-plugin-formula/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": "../../tsconfig.json",
+    "compilerOptions": {
+        "rootDir": "./",
+        "outDir": "lib",
+        "baseUrl": "./",
+        "esModuleInterop": true, // auto allowSyntheticDefaultImports true
+        "strictPropertyInitialization": false
+    },
+    "include": [
+        "src/**/*",
+        "test/**/*.test.(ts|tsx|js)"
```

Comment:
All test files are located in the `src` directory, so this line may be unnecessary.